packages/@sanity/components/src/dialogs/DefaultDialog.js

```diff
@@ -4,19 +4,22 @@ import CloseIcon from 'part:@sanity/base/close-icon'
 import styles from 'part:@sanity/components/dialogs/default-style'
 import Button from 'part:@sanity/components/buttons/default'
 import Portal from 'react-portal'
-import StackedEscapable from '../utilities/StackedEscapable'
+import Escapable from '../utilities/Escapable'
+import CaptureOutsideClicks from '../utilities/CaptureOutsideClicks'
+import Stacked from '../utilities/Stacked'
 
 export default class DefaultDialog extends React.PureComponent {
   static propTypes = {
     kind: PropTypes.oneOf(['default', 'warning', 'success', 'danger', 'info']),
     className: PropTypes.string,
-    title: PropTypes.string.isRequired,
+    title: PropTypes.string,
     children: PropTypes.node,
     onOpen: PropTypes.func,
     onClose: PropTypes.func,
     isOpen: PropTypes.bool,
     onAction: PropTypes.func,
     showHeader: PropTypes.bool,
+    showCloseButton: PropTypes.bool,
```

Comment:
Could we use the `onClose`-prop to signal whether to show the close button or not? E.g. only show it if `onClose` is passed? It doesn't make sense to pass this as `true` and no `onClose`, right?

Reply:
I had the same thought. But the design team wants actions in the dialog-footer to have the close action. If using the dialog without actions, it should be closable as default.

Reply:
I see 👍